### ISSUE TITLE
Medic Call: Swearing does not play properly with Brundle Bundle

### ISSUE THREAD

**Issue by Zane-Truesdale** [redsunservers/LoadoutBugTracker#381]
Title. While all other saysound replacements I have used with the Brundle Bundle (the fly engineer set) have played just fine, Swearing does not play it correctly when the set is active. I think it might be starting from the middle of the voice line based on how it sounds as it stops at Bit-

I have provided a video clip below to show what I mean, as well as a comparison to the normal Swearing call to show the difference
[Team Fortress 2 2019-06-02 19-42-53_Trim.zip](https://github.com/redsunservers/LoadoutBugTracker/files/3245398/Team.Fortress.2.2019-06-02.19-42-53_Trim.zip)

**Comment by FortyTwoFortyTwo** [comment]
Me and Zane taken a look at it, really unsure how it happens when it should work fine from debugs, and how it works for any other voice replacements, using same code for it.

With all of that, it only affects swearing with engi fly set equipped, so not too much of a deal in the end anyway. Not worth fixing it in the moment.

**Comment by Zane-Truesdale** [comment]
Yeah, it was very confusing as to why everything else plays fine but this one has issues. Also worth noting that voice pitch alterations via baby coins without the fly set on does not cause this issue.

tl;dr Very confusing matter for a miniscule voice bug. +1 the closing
